server/src/main/java/org/apache/iotdb/db/metadata/schemaregion/ISchemaRegion.java

```diff
@@ -135,12 +135,13 @@ Pair<Integer, Set<String>> deleteTimeseries(PartialPath pathPattern, boolean isP
    * path, may contain wildcard. If using prefix match, the path pattern is used to match prefix
    * path. All timeseries start with the matched prefix path will be counted.
    */
-  int getAllTimeseriesCount(PartialPath pathPattern, boolean isPrefixMatch)
+  int getAllTimeseriesCount(PartialPath pathPattern, boolean isPrefixMatch, boolean hasTag)
       throws MetadataException;
 
   // The measurements will be grouped by the node in given level and then counted for each group.
   Map<PartialPath, Integer> getMeasurementCountGroupByLevel(
-      PartialPath pathPattern, int level, boolean isPrefixMatch) throws MetadataException;
+      PartialPath pathPattern, int level, boolean isPrefixMatch, boolean hasTag)
+      throws MetadataException;
```

Comment:
What about extend a interface with same name but override it? It means keep the old method since it has been used widely, and use the new method to support new feature? I think this makes the modification more clear.